packages/suite/src/components/suite/notifications/NotificationRenderer/NotificationRenderer.tsx

```diff
@@ -229,6 +230,17 @@ export const NotificationRenderer = ({
                     }}
                 />
             );
+        case 'tx-exchange': {
+            return (
+                <ExchangeInfoRenderer
+                    render={render}
+                    notification={notification}
+                    icon="arrowUp"
+                    variant="info"
```

Comment:
```suggestion
                    variant="success"
```

let's use `success` for now before pending/mined states are introduced